Sources/CgRPC/src/core/ext/filters/http/client/http_client_filter.c

```diff
@@ -505,9 +505,7 @@ static grpc_slice user_agent_from_args(const grpc_channel_args *args,
     }
   }
 
-  gpr_asprintf(&tmp, "%sgrpc-c/%s (%s; %s; %s)", is_first ? "" : " ",
-               grpc_version_string(), GPR_PLATFORM_STRING, transport_name,
-               grpc_g_stands_for());
+  gpr_asprintf(&tmp, "");
```

Comment:
Modifications to the gRPC-Core code will be overwritten by any updates to the vendored code, and are not included in the CocoaPods distribution of the library (because that uses the public gRPC-Core repo directly). Is there another way to accomplish this?

Reply:
Yeah, I agree, we shouldn't overwrite `CgRPC`. My short term goal is to make `grpc-swift` available through Carthage and we need to use libraries like `gRPC-Core`, `CZlib`, `BoringSSL` as dependencies. I'll look into it and see if there is another way but all my attempts so far weren't concluding.